gce/image/build_image.sh

```diff
@@ -138,9 +109,6 @@ if [ ! -f variables.json ]; then
     exit 1
 fi
 
-cd $DIR
-mkdir -p build
```

Comment:
Why removing the mkdir, we need to create the build directory, otherwise it would fail

Reply:
mistake, it's back now